x-pack/plugins/apm/server/routes/mobile/get_mobile_stats.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  termQuery,
+  kqlQuery,
+  rangeQuery,
+} from '@kbn/observability-plugin/server';
+import { ProcessorEvent } from '@kbn/observability-plugin/common';
+import {
+  SERVICE_NAME,
+  TRANSACTION_TYPE,
+  SESSION_ID,
+  SPAN_SUBTYPE,
+  APP_LAUNCH_TIME,
+  EVENT_NAME,
+} from '../../../common/es_fields/apm';
+import { environmentQuery } from '../../../common/utils/environment_query';
+import { APMEventClient } from '../../lib/helpers/create_es_client/create_apm_event_client';
+import { getBucketSize } from '../../lib/helpers/get_bucket_size';
+
+type Timeseries = Array<{ x: number; y: number | null }>;
+export interface MobileStats {
+  sessions: { value?: number; timeseries: Timeseries };
+  requests: { value?: number | null; timeseries: Timeseries };
+  maxLoadTime: { value?: number | null; timeseries: Timeseries };
+  crashCount: { value?: number | null; timeseries: Timeseries };
+}
+
+export async function getMobileStats({
+  kuery,
+  apmEventClient,
+  serviceName,
+  transactionType,
+  environment,
+  start,
+  end,
+}: {
+  kuery: string;
+  apmEventClient: APMEventClient;
+  serviceName: string;
+  transactionType?: string;
+  environment: string;
+  start: number;
+  end: number;
+}): Promise<MobileStats> {
+  const { intervalString } = getBucketSize({
+    start,
+    end,
+    minBucketSize: 60,
+  });
+
+  const aggs = {
+    sessions: {
+      cardinality: { field: SESSION_ID },
+    },
+    requests: {
+      filter: { term: { [SPAN_SUBTYPE]: 'http' } },
+    },
+    maxLoadTime: {
+      max: { field: APP_LAUNCH_TIME },
+    },
+    crashCount: {
+      filter: { term: { [EVENT_NAME]: 'crash' } },
+    },
```

Comment:
Can you try using the `rate` agg?

```suggestion
    crashCount: {
      filter: { term: { [EVENT_NAME]: 'crash' } },
      aggs: { rate: { unit: minute }}
    },
```

Reply:
Tried to use `rate` agg but I couldn't use it with filtering the `event.name: 'crash'`

@dgieselaar any ideas on how I could better use rate agg to get the crashes per minute? 

Now I'm getting the crash count and dividing by duration As Minutes,